main/lsp/wrapper.cc

```diff
@@ -84,6 +84,15 @@ LSPWrapper::LSPWrapper(string_view rootPath, bool disableFastPath) {
     instantiate(std::move(gs), logger, disableFastPath);
 }
 
+LSPWrapper::LSPWrapper(options::Options &&options, std::string_view rootPath, bool disableFastPath)
+    : opts(std::move(options)) {
+    instantiate(rootPath, disableFastPath);
+}
+
+LSPWrapper::LSPWrapper(string_view rootPath, bool disableFastPath) {
```

Comment:
Why not just dispatch to constructor with more args in the initialize, instead of sharing behavior in `initialize`?

Reply:
Sorry, I meant in the constructor body, not in the initializer list :)

Reply:
Whoops; I forgot about this language feature... Good point!